### ISSUE TITLE
generating imbedding 

### ISSUE THREAD

**Issue by halimaaol** [nitishajain/KGESemanticAnalysis#2]
Hi,
I want to generate embeddings from FB15k dataset using pyrdf2vec, how did you generate the embeddings from tsv file?

**Comment by JanKalo** [comment]
I think, I do not have the code for it anymore. But it should be possible to follow the example on their GitHub repo: https://github.com/IBCNServices/pyRDF2Vec

**Comment by halimaaol** [comment]
if you still have the code please share it with me

**Comment by JanKalo** [comment]
No, like I said. We do not have the Code for training the model. But we were just using the standard RDF2Vec code that is described in the example on the github repo.

**Comment by halimaaol** [comment]
@JanKalo thank you
